mkdocs.yml

```
@@ -93,6 +93,53 @@ plugins:
         'unified/novacustom/fan-profiles.md': 'unified/novacustom/features.md'
         'unified/novacustom/fn-lock-hotkey.md': 'unified/novacustom/features.md'
         'unified/novacustom/rgb-keyboard.md': 'unified/novacustom/features.md'
+
+        'variants/protectli_fw6/building_manual.md': 'unified/protectli/building_manual.md#fw6'
+        'variants/protectli_vp1x1x/building_manual.md': 'unified/protectli/building_manual.md#v1000-series'
```

```suggestion
        'variants/protectli_v1x10/building_manual.md': 'unified/protectli/building_manual.md#v1000-series'
```

The old links were v1x10, not vp1x1x, please fix in all the redirects

---

https://github.com/Dasharo/docs/pull/851/commits/18bed89c79cc499bf592fe93f0559f083dc1d6fc